src/components/hv-section-list/index.js

```diff
@@ -109,7 +109,7 @@ export default class HvSectionList extends PureComponent<
           this.props.onUpdate,
           this.props.options,
         ),
-      sections,
+      sections: sections.filter(s => s.data.length > 0),
```

Comment:
There may be empty sections for which we do want to show just the headers. But it may not have been rendering correctly anyway so we should handle those differently.

Reply:
| may not have been rendering correctly anyway

What does this mean? Is a section with a header but no data actually causing problems?

Just curious, did you also try an approach where we render an item with an empty view if we don't have data.